### PR TITLE
Progress bar added

### PR DESCRIPTION
**Pull request by allangoncalves**
Added a progress bar so the user can see the percentage of the build update and the time spent while doing it.

### CODE REVIEW

**Comment by LTKills** [comment]
Thank you for comtributing, but unfortunately I cannot accept this PR. Every PR should tackle an specific issue and the contributos should be assigned to the issue for the PR to be accepted. Please open an issue about It and read the contributing section on README

File: src/heroes_build_scrapper/data.py [hunk 1]
```diff
@@ -31,12 +32,15 @@ def update_heroes_list():
 def update_all_builds():
     '''Updates all builds of all heroes (calls update_builds for all heroes)
     '''
-    print('Starting full heroes build update')
+    print('Updating Heroes')
```
Comment (LTKills) [inline src/heroes_build_scrapper/data.py]: :bike: (bike means optional)
I prefer the original phrasing of this

File: src/heroes_build_scrapper/data.py [hunk 2]
```diff
@@ -31,12 +32,15 @@ def update_heroes_list():
...
 
     with open('data/heroes.json', 'r') as fp:
         heroes = json.load(fp)
 
-    for hero in heroes:
+    bar = tqdm(heroes, bar_format='{l_bar}{bar}{elapsed}')
+    for hero in bar:
+        bar.set_description("{0:^16}".format(hero))
         update_hero_builds(hero)
+
```
Comment (LTKills) [inline src/heroes_build_scrapper/data.py]: :scissors:

**Review by gmelodie — CHANGES_REQUESTED** [review]
Looks good, be sure to add your github nick on `PLAYERS.md` as well

**Comment by allangoncalves** [comment]
The old updating message is back! :)

**Comment by LTKills** [comment]
I commented the :scissors:, please recheck that

**Pull request closed without merging**